let(var a = 10, var b = a ^ 3)
{
    var x = a + b;
    println(x);
};


// Expected to fail as x is valid only inside 'let' context
print(x);
